eslint-config-landr/package.json

```diff
@@ -30,5 +30,8 @@
     },
     "engines": {
         "node": ">= 4"
+    },
+    "dependencies": {
+        "eslint-plugin-react": "^7.16.0"
     }
```

Comment:
we should not add this in dependencies if we don't use react eslint.
Also it forces a version, I think we should let the user choose.
You can use `optionalDependencies` or `peerDependencies` for this

Reply:
true